### ISSUE TITLE
How to generate a single image after training

### ISSUE THREAD

**Issue by PiyKat** [carpedm20/DCGAN-tensorflow#224]
Hello,

I have been trying to play with the code a little bit with the intention of creating just a single image instead of an 8X8 grid but to no avail. Can somebody point out how to do the same.

**Comment by xx-zhou16** [comment]
I have the same problem.

**Comment by PiyKat** [comment]
@xx-zhou16 A temporary workaround is that you can return the avergage of the grid of pictures rather than the grid itself. If you would like to do that, you can change line 429 in model.py to  `np.mean(tf.nn.tanh(h4),axis=0).`
This will cause information loss but if you train it for multiple epochs it will get better.

**Comment by BLCKPSTV** [comment]
Is there a working solution for this? This must be availble not, if the demo show a single image. It is already accomplished by one of the makers? @PiyKat

**Comment by benckx** [comment]
I created a fork with these two additional parameters: `grid_height` and `grid_width` that can be used both during training and testing. They override `batch_size` (`batch_size = grid_height * grid_width`).

https://github.com/benckx/DCGAN-tensorflow#about-this-fork

**Comment by minxdragon** [comment]
I found adding `--batch_size=1 --option=1` to my code produces only one image. there are other options for outputting gifs too (found in utils.py). but I haven't experimented with them yet.
